src/views/Conference.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <Page class="league-page-container">
+    <div
+      class="league-page"
+    >
+      <template v-if="counter > 0">
+        {{ counter }}
+      </template>
+      <template v-else>
+        <div
+          id="jitsi"
+        />
+      </template>
+    </div>
+  </Page>
+</template>
+<script>
+// eslint-disable-next-line import/no-extraneous-dependencies
+import JitsiMeetExternalAPI from 'jitsi';
+import { mapState, mapGetters, mapMutations } from 'vuex';
+import Page from '../components/layout/Page.vue';
+import Util, { IS_MOBILE_DEVICE, createDeepLinkUrl } from '../utils/util';
+import { client } from '../utils/aeternity';
+
+const TIMEOUT = 400000;
```

Comment:
This is 400 sec, though.

Reply:
it's not used at all. but I will remove that.